lib/integration/queryManager.ts

```diff
@@ -0,0 +1,30 @@
+import { logMessage } from "../logger";
+import dbConnector from "./dbConnector";
+import { QueryArrayResult } from "pg";
+
+const queryManager = {
```

Comment:
Why is this in an object ? Just export each individually

Reply:
In order to mock these functions  i either have to create to separate module  (file) which i found heavy for this purpose.So i create a namespace instead.

Reply:
Considering you only have one function now you don't need this. Also to mock individual functions you don't need to create a separate file. You can just `jest.mock()` the module and you can define the mock behaviour in the individual tests. See the following section https://jestjs.io/docs/mock-functions#mocking-modules